ansible/management-searchengine.yml

```diff
@@ -0,0 +1,253 @@
+# Search ngine + Search Engine Client + Elasticsearch
+
+#- hosts: "{{ idr_environment | default('idr') }}-management-hosts"
+
+ 
+#hosts: localhost
+#connection: local
+
+#- hosts: all
+#  remote_user: root
+#  vars_files:
+#    - /vars/external_vars.yml
+#- hosts: all
+#  vars_files:
+#    file: searchengine_vars.yml
+
+- name: Deploying search engine
+  connection: local
+  hosts: local
+  vars_files:
+    searchengine_vars.yml
+
+  pre_tasks:
+    - raw: sudo yum update -y
```

Comment:
At the moment, the first playbook executed after the provisioning (https://github.com/IDR/deployment/blob/master/ansible/idr-00-preinstall.yml#L7) upgrade all the packages before installing the services. So this should be unnecessary and can be removed.

Reply:
Ok, I have removed it and removed the link from the playbooks.